console/src/declaration.d.ts

```diff
@@ -6,3 +8,8 @@ declare module '*.scss' {
 declare namespace SvgPanZoom {
   export interface Instance {}
 }
+declare module 'react' {
+  interface HTMLAttributes<T> extends DOMAttributes<T> {
+    css?: CSSProp;
+  }
+}
```

Comment:
It is fix this type error.

https://github.com/hasura/graphql-engine/pull/4504/commits/21a64fdf7adf07f6f2522a27f1411de7a645e92b#diff-2045ac77f10a7a2f5b039fa2460c841eR56